anchore_engine/db/entities/policy_engine.py

```diff
@@ -2599,14 +2601,14 @@ def fixed_artifact(self):
 
         return fixed_in
 
-    def fixed_in(self):
+    def fixed_in(self, fixed_in: FixedArtifact = None):
```

Comment:
This code makes sense, but we should review later the inclusion of these functions in the db entity types. I think we're going to have to decouple them, but let's make a tech-debt issue for that and track that for all policy engine entities.

Reply:
Shouldn't this formatting just be put on the FixedArtifact class to simplify the flow?

Reply:
I agree that this should not be a part of the ImagePackageVulnerability entity code but this will all go away when we get rid of those persistence models as we deprecate the legacy vulnerability matching in policy-engine. Addition of the new argument is to avoid computing the fixed artifact multiple times, I know it's hacky